packages/fxa-auth-server/lib/senders/emails/layouts/subscription/index.scss

```diff
@@ -28,6 +28,10 @@
   @extend .mb-6;
 }
 
+.text-body li {
```

Comment:
We could add a comment above this with a short reason why we need it. Otherwise someone might remove this later because they don't see any visual difference.